Sources/Bitset/Bitset.swift

```diff
@@ -225,22 +211,22 @@ public final class Bitset: Sequence, Equatable, CustomStringConvertible,
 
   // compute the intersection (in place) with another bitset
   public func intersection(_ other: Bitset) {
-    let mincount = other.wordcount < self.wordcount ? other.wordcount : self.wordcount
+    let mincount = Swift.min(self.wordcount, other.wordcount)
```

Comment:
Any particular reason for using "Swift.min" as opposed to just "min"?

Reply:
the compiler complains because min refers to the the min() method.